packages/accordion-react/src/Accordion.tsx

```diff
@@ -1,12 +1,12 @@
 import React, { FC } from "react";
 
-interface Props {
+export interface AccordionProps {
```

Comment:
Nice 👍

For at typer skal kunne importeres lettere er det fint om du også endrer [accordion-react/index.ts](https://github.com/fremtind/jokul/blob/main/packages/accordion-react/src/index.ts) til dette:

```ts
export type { AccordionItemProps } from "./AccordionItem";
export { AccordionItem } from "./AccordionItem";
export type { AccordionProps } from "./Accordion";
export { Accordion } from "./Accordion";
```

Reply:
takk, @wkillerud! Har løst det her: 4949d3c95b2a2c4ffeb028c5afe701f36912bb11